LoopKit/DeviceManager/CGMManager.swift

```diff
@@ -71,6 +74,12 @@ public protocol CGMManager: DeviceManager {
     func fetchNewDataIfNeeded(_ completion: @escaping (CGMResult) -> Void) -> Void
 }
 
+public extension CGMManager {
```

Comment:
We generally try to avoid default implementations for protocols.  See https://medium.com/@xcadaverx/default-protocol-extension-caveats-88c6ece27fdb 

I think this should be removed, which means that implementations will need to be added for the existing CGM managers.